home/dot_config/mackup/.mackup.cfg

```diff
@@ -0,0 +1,79 @@
+[storage]
+engine = file_system
+path = .local
+directory = mackup
+
+[applications_to_sync]
+aria2c
+autokey
+aws
+azure
+bash
+bat
+beets
+brave
+cerebro
+chrome
+composer
+conky
+docker
+edge
+emacs
+ferdium
+filezilla
+firefox
+fontconfig
+gimp
+git
+git-hooks
+git-kraken
+gradle
+homebrew
+htop
+httpie
+hyper
+inkscape
+intellijidea
+iterm2
+kitty
+kubectl
+libreoffice
+librewolf
+littlesnitch
+lollypop
+mackup
+microsoft-remote-desktop
+mitmproxy
+ngrok
+npm
+npmrc
+pandoc
+pass
+pip
+pnpm
+poetry
+powerline
+prezto
+rbenv
+remmina
+rclone
+ripgrep
+ruby
+rvm
```

Comment:
I don't think we're using rvm -- we're using asdf for ruby version management

Reply:
Removed `rvm` and some other that have been replaced. There may be some more to remove, please review.